Bottom: 66ef3f3a0d6c6d45cead1bd9a045f9fe37cc000b
Top:    fd88bc0d719eda1b7ca07e6edc4db1d65f100d9b
Author: scott snyder <sss@karma>
Date:   2018-08-26 20:24:30 -0400

Bump target version to 20.



---

diff --git a/ComicReader/project.properties b/ComicReader/project.properties
index ea89160..92b1625 100755
--- a/ComicReader/project.properties
+++ b/ComicReader/project.properties
@@ -8,4 +8,4 @@
 # project structure.
 
 # Project target.
-target=android-8
+target=android-20
